lang/en.json

```diff
@@ -823,6 +823,15 @@
 "SETTINGS.AutoConfirmSelf": "Self-Target Actions Only",
 "SETTINGS.AutoConfirmAll": "Auto-Confirm All Actions",
 
+"SETTINGS.AncestrySourcesName": "Ancestry Sources",
+"SETTINGS.AncestrySourcesHint": "Which packs will be used as sources of ancestries during character creation.",
```

Comment:
1. Field labels and hints should canonically use lowercase `label` and `hint` as the final parts of their localization keys. This allows them to (potentially, but not required) work with prelocalization.
2. Let's use a bit of nesting rather than keeping the keys flat. For example:
```js
"SETTINGS": {
  "COMPENDIUM_SOURCES": {
    "ancestry": {
      "label": "Ancestry Sources",
      "hint": "Which packs will be used as sources of ancestries during character creation."
    },
    ...
  }
}
```